src/Components/Common/ClosingTag.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+function ClosingTag() {
```

Comment:
For Icon, You can use
- Material icons
- Font awesome icons

Reply:
made the changes